ansible/roles/zuul/tasks/main.yml

```diff
@@ -49,6 +49,15 @@
       src: '{{ config_dir }}/zuul/'
       dest: /etc/zuul/
 
+- name: 'retrieve ip address of zuul host'
+  set_fact:
+     zuul_host: '{{ ansible_default_ipv4.address }}'
+
+- name: 'generate zuul.conf from template'
+  template:
+     src: {{ config_dir }}/zuul.conf.j2
+     dest: /etc/zuul
```

Comment:
Prefer using trailing slash when copying to a directory, ie: `/etc/zuul/`

Reply:
Actually, I'm not even sure if `dest` can be a directory when using a template, I'd go with a full path here.